Classes/Utility/ColorSpace.php

```diff
@@ -31,7 +31,7 @@ class ColorSpace
      * @return string
      * @throws \InvalidArgumentException
      */
-    public static function normalize($str)
+    public static function normalize(string $str = null): ?string
```

Comment:
cool. nullable, but in that case not TYPO3 8 compatible anymore. Would be OK for me.

Reply:
> cool. nullable, but in that case not TYPO3 8 compatible anymore. Would be OK for me.

Why would it not be TYPO3 8 compatible with a nullable type? This is available since PHP 7.1, so the only thing to do is dropping compatibility with PHP 7.0, nothing directly related to TYPO3 v8 since TYPO3 v8 works fine with TYPO3 7.2 or 7.3.

Reply:
raised version